helm/reana/values.yaml

```diff
@@ -66,17 +66,17 @@ components:
       threads: 4
   reana_workflow_controller:
     imagePullPolicy: IfNotPresent
-    image: reanahub/reana-workflow-controller:0.7.1
+    image: reanahub/reana-workflow-controller:0.7.2
     environment:
       SHARED_VOLUME_PATH: /var/reana
   reana_workflow_engine_cwl:
-    image: reanahub/reana-workflow-engine-cwl:0.7.2
+    image: reanahub/reana-workflow-engine-cwl:0.7.3
   reana_workflow_engine_yadage:
-    image: reanahub/reana-workflow-engine-yadage:0.7.2
+    image: reanahub/reana-workflow-engine-yadage:0.7.3
   reana_workflow_engine_serial:
-    image: reanahub/reana-workflow-engine-serial:0.7.2
+    image: reanahub/reana-workflow-engine-serial:0.7.3
   reana_job_controller:
-    image: reanahub/reana-job-controller:0.7.2
+    image: reanahub/reana-job-controller:0.7.3
```

Comment:
Replace with `reana-job-controller-htcondorcern-slurmcern:0.7.3` to test HTCondor and Slurm.